h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -613,13 +613,15 @@ public Vec cv_AssignFold(int N) {
     for( int i=0; i<N; i++ ) {
       String identifier = origDest + "_cv_" + (i+1);
       // Training/Validation share the same data, but will have exclusive weights
-      Frame cvTrain = new Frame(Key.<Frame>make(identifier+"_train"),cv_fr.names(),cv_fr.vecs());
+      Frame cvTrain = new Frame(Key.make(identifier+"_train"),cv_fr.names(),cv_fr.vecs());
```

Comment:
since you are changing this I would add the missing spaces to improve readability